File: web/src/task-force/models/TaskForce.ts
export interface TaskForce {
    id: string;
    name: string;
    robots: Array<string>;
}

export class TaskForceModel {
    //@ts-ignore
    private _object: TaskForce;

    get id(): string {
        return this._object.id;
    }

    get name(): string {
        return this._object.name;
    }

    get robots(): Array<string> {
        return this._object.robots
    }
}

File: core/src/main/java/org/robotframework/roc/core/models/TaskForce.java
package org.robotframework.roc.core.models;

import lombok.Data;
import org.springframework.data.redis.core.RedisHash;

import java.io.Serializable;

@Data
@RedisHash("TaskForce")
public class TaskForce implements Serializable {

    private Long id;
    private Long projectId;
    private String name;

    private String robots;

}
